styles/desktop.css

```diff
@@ -1,5 +1,4 @@
 
-
 * {
   margin: 0;
```

Comment:
![5](https://user-images.githubusercontent.com/77843050/221177852-2c112c0e-7c16-4a45-a6f7-698cbd82034c.gif)

- [ ] Hey @RHK-MICROVERSE, I like your user name,😄... so you have created the desktop version CSS version but you are missing the media query in it, regarding the requirement, you should have it from `768px`, please wrap the whole desktop CSS inside a media query, fix all the necessary broken parts and resubmit your work.

<img width="919" alt="1" src="https://user-images.githubusercontent.com/77843050/221179162-00707c4c-51ba-47fa-9922-b2455aefa20a.png">